src/main/java/uk/gov/companieshouse/efs/api/email/mapper/DelayedSubmissionSupportEmailMapper.java

```diff
@@ -48,6 +48,7 @@ private DelayedSubmissionSupportEmailData fromDelayedSubmissions(DelayedSubmissi
                 .withTo(config.getSupportEmailAddress())
                 .withSubject(config.getSubject())
                 .withDelayedSubmissions(model.getDelayedSubmissions())
+                .withThresholdInMinutes(model.getThresholdInMinutes())
```

Comment:
do we need the Minutes in the existing email?

Reply:
ah yes I think I see, we don't have a separate 'Sameday' model.  Can you just confirm?

Reply:
Yes, we need the minutes value for the same day but not the standard email.